docs/welcome.md

```diff
@@ -17,7 +17,7 @@ This page is provided as the entrypoint to the different topics of this user-gui
 
 ## Developer
 
-- Start contributing: [Appendix/Contributing](appendix/contributing)
+- Start contributing: [Appendix/Contributing](appendix/contributing.md)
```

Comment:
Please change to `../appendix/contributing`.  This creates an internal link.

We don't want to link out just to redirected back in.

Reply:
@mazzystr Thanks for the review.
I have updated it as suggested.